guides/release/tutorial/autocomplete-component.md

```diff
@@ -79,25 +79,24 @@ The `handleFilterEntry` action will apply the search term filter to the list of
 
 ```javascript {data-filename=app/components/list-filter.js}
 import Component from '@ember/component';
+import { action } from '@ember-decorators/object';
 
-export default Component.extend({
-  classNames: ['list-filter'],
-  value: '',
+export default class ListFilterComponent extends Component {
```

Comment:
Not to muddy the waters here too much, but I thought native class syntax should only apply to Glimmer components: https://github.com/ember-learn/guides-source/blob/octane/guides/release/upgrading/editions.md#glimmer-components. Does this seem correct?

Reply:
@pzuraq can you review this bit please?

Reply:
yes, native class syntax applies to everything _except_ classic components. This is because we don't have first class APIs for things like `classNames`.

Reply:
The recent changes to the component blueprints in emberjs/ember.js to support octane were done _before_ we had decided that we didn't want to show native class syntax with `Ember.Component` at all. 

Would someone mind making an issue over there to get it fixed (if you cc @ppcano he will probably be willing to fix it up...)

Reply:
created that issue: https://github.com/emberjs/ember.js/issues/17701